src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -16,44 +17,70 @@ import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
 import { PLACEHOLDER_USER } from '../../../../constants';
 import type { Annotation, AnnotationPermission } from '../../../../common/types/feed';
 import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
-import type { User } from '../../../../common/types/core';
+import type { SelectorItems, User } from '../../../../common/types/core';
 
 import './AnnotationActivity.scss';
 
 type Props = {
     currentUser?: User,
     getAvatarUrl: GetAvatarUrlCallback,
+    getMentionWithQuery?: Function,
     getUserProfileUrl?: GetProfileUrlCallback,
     isCurrentVersion: boolean,
     item: Annotation,
+    mentionSelectorContacts?: SelectorItems<User>,
     onDelete?: ({ id: string, permissions: AnnotationPermission }) => any,
+    onEdit?: (id: string, text: string, permissions: AnnotationPermission) => any,
     onSelect?: (annotation: Annotation) => any,
 };
 
 const AnnotationActivity = ({
+    currentUser,
     item,
     getAvatarUrl,
+    getMentionWithQuery,
     getUserProfileUrl,
     isCurrentVersion,
+    mentionSelectorContacts,
     onDelete = noop,
+    onEdit = noop,
     onSelect = noop,
 }: Props) => {
+    const [isInputOpen, setIsInputOpen] = React.useState(false);
+    const [isEditing, setIsEditing] = React.useState(false);
     const { created_at, created_by, description, error, file_version, id, isPending, permissions = {}, target } = item;
+    const message = (description && description.message) || '';
```

Comment:
Why move this line up here?

Reply:
I can move it back down, I moved it up before because it was being used by one of the handlers.